org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/ui/editor/EditorFragment2.xtend

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2017 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.xtext.generator.ui.editor
+
+import com.google.inject.Inject
+import org.eclipse.xtext.xtext.generator.AbstractStubGeneratingFragment
+import org.eclipse.xtext.xtext.generator.XtextGeneratorNaming
+import org.eclipse.xtext.xtext.generator.model.FileAccessFactory
+import org.eclipse.xtext.xtext.generator.xbase.XbaseUsageDetector
+
+/**
+ * @author fstolte - Initial contribution and API
```

Comment:
`@since 2.14` shhould be exported in manifest

Reply:
I added the package export to the manifest.

Reply:
@since is still missing @FStolte

Reply:
Oh right, I missed that. @Since is now added.